waiter/src/waiter/token.clj

```diff
@@ -437,9 +443,9 @@
 (defn- handle-post-token-request
   "Validates that the user is the creator of the token if it already exists.
    Then, updates the configuration for the token in the database using the newest password."
-  [clock synchronize-fn kv-store cluster-calculator token-root history-length limit-per-owner waiter-hostnames
+  [clock custom-components synchronize-fn kv-store cluster-calculator token-root history-length limit-per-owner waiter-hostnames
    entitlement-manager make-peer-requests-fn validate-service-description-fn attach-service-defaults-fn
-   tokens-update-chan {:keys [headers] :as request}]
+   tokens-update-chan post-validator-fn {:keys [headers] :as request}]
```

Comment:
When `post-validator` is a component, then the `custom-components` will not need to be passed as an argument to this function.

Reply:
Thanks, that will be a cleaner change